### PR TITLE
Techno helmet

### PR DESCRIPTION
**Pull request by Ayesha-678**
Adds a techno helmet, and a hypnotic spiral background.

### CODE REVIEW

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -4404,6 +4404,20 @@ var AssetFemale3DCG = [
 			},
 			{ Name: "OpenFaceHood", Fetish: ["Latex"], Value: 35, Priority: 40, Difficulty: 5, Time: 15, AllowLock: false, DefaultColor: "#404040", BuyGroup: "OpenFace", Block: ["ItemEars"], Hide: ["HairFront", "HairBack", "Ears"], HideItem: ["HatFurHeadband"]},
 			{ Name: "GwenHood", Fetish: ["Leather"], Value: 35, Difficulty: 5, Time: 15, AllowLock: true, DefaultColor: "#404040", Hide: ["HairFront", "Ears",], Block: ["ItemMouth", "ItemMouth2", "ItemMouth3", "ItemEars"], Extended: true, Alpha: [{ Group: ["ItemMouth", "ItemMouth2", "ItemMouth3"], Masks: [[0, 185, 400, 400]] }], HideItem: ["HatFurHeadband"], HasType: false },
+			{ Name: "TechnoHelmet1", Category: ["SciFi"], Fetish: ["Metal"], Value: 100, Difficulty: 7, DefaultColor: ["#1E7484", "Default", "#2F2F2F", "#999999", "#246489"], RemoveTime: 10, AllowLock: true, DrawLocks: false, Effect: [], Hide: ["HairFront", "Ears", "HairBack", "HairAccessory1", "HairAccessory2", "HairAccessory3", "ItemEars", "Glasses"], Extended: true,
+		        Layer: [
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:4408]:
```suggestion
				Layer: [
```

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 2]
```diff
@@ -4404,6 +4404,20 @@ var AssetFemale3DCG = [
...
+					{ Name: "LightlyTintedVisor", HasType: false, CopyLayerColor: "TransparentVisor", AllowModuleTypes: ["v2"] },
+					{ Name: "HeavilyTintedVisor", HasType: false, CopyLayerColor: "TransparentVisor", AllowModuleTypes: ["v3"] },
+					{ Name: "OpaqueVisor", HasType: false, CopyLayerColor: "TransparentVisor", AllowModuleTypes: ["v4", "v5"] },
+					{ Name: "HypnoticVisor", HasType: false, CopyLayerColor: "TransparentVisor", AllowModuleTypes: ["v5"] },
+					{ Name: "HypnoSpiral", HasType: false, AllowModuleTypes: ["v5"] },
+					{ Name: "Shine", HasType: false, AllowColorize: false, AllowModuleTypes: ["v1", "v2", "v3", "v4", "v5"] },
+					{ Name: "ChinStrap", HasType: false, AllowModuleTypes: ["c1"] },
+					{ Name: "HelmetMain", HasType: false, },
+					{ Name: "HelmetSecondary", HasType: false, },
+				]
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:4419]:
```suggestion
				]
```

**Review by ace-1331 — DISMISSED** [review]

**Review by ace-1331 — CHANGES_REQUESTED** [review]
- missing the HypnoSpiral layer name
- when in hypno mode, Im getting a missing asset error: TechnoHelmet1_HypnoticVisor.png  is it missing or should that layer have no image?

**Comment by Shiranui-Izayoi** [comment]
Maybe make the helmet cover a bit more of the forehead. As I feel it exposes too much, causing it to look off a bit.

**Comment by Ben987** [comment]
Right on time for the next beta, we can always adjust the details later.  Thanks a lot!

**Pull request merged**